_projects/Obstacle_Search

```diff
@@ -0,0 +1,11 @@
+---
+layout: project
+title: Obstacle_Search
+desc: |
+ The final project for obstacle_search. It is a "temple run"-like game that train player how to respond to obstacles using three different modes.
+dl: https://drive.google.com/file/d/1HQOH30LV6yaHUS44NnB7oANnuVkBq9Ln/view?usp=sharing
+semester: Spring 2019
+team: 04
+youtube: KodswdrBYVk
+featured: false
+---
```

Comment:
Do you want to explain your motivation, and the professor/lab that you worked with? There's more to add in this description.